js/form_validation.js

```diff
@@ -81,7 +83,15 @@ form.addEventListener('change', (evt) => {
 
 form.addEventListener('submit', (evt) => {
   evt.preventDefault();
-  pristine.validate();
+  const isValid = pristine.validate();
+
+  if (isValid) {
+    blockSubmitButton();
+    (sendOfferToServer(sendOfferSuccess, sendOfferError, new FormData(evt.target)))
+      .then(() => {
+        evt.target.reset();
+      });
```

Comment:
```suggestion
      .then(evt.target.reset);
```

ну можно же передать ссылку на метод?) А еще лучше сделай деструктуризацию из evt и получи ссылку на метод ресет